config/purposes/scrna_core_cdna_prep.wip.yml

```diff
@@ -112,3 +112,73 @@ LRC HT 5p cDNA PCR XP:
   :asset_type: plate
   :stock_plate: false
   :input_plate: false
+# TODO: there are more steps in the pipeline before this cherrypick plate.
+LRC HT 5p Cherrypick:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :cherrypickable_target: true
+# NB. the min and max amounts for the bins are the total amount of cDNA in the well i.e.
+# QC concentration * (source_volume + diluent_volume)
+# TODO: R&D to confirm the ranges for each PRC cycle
+LRC HT 5p cDNA Dil:
+  :asset_type: plate
+  :stock_plate: false
+  :input_plate: false
+  :presenter_class: Presenters::ConcentrationBinnedPlatePresenter
+  :creator_class: LabwareCreators::ConcentrationBinnedSingleCellDilPlate
+  :dilutions:
+    :source_volume: 20
+    :diluent_volume: 0
+    :bins:
+      - max: 5.0
+        pcr_cycles: 16
+        colour: 1
+      - min: 5.0
```

Comment:
I think the idea was to ensure you get the same colour (for say 14 PCR cycles) each time, rather than it dynamically assigning numbers depending on the binning range in that particular plate, so you'd need some config or table somewhere that persisted that relationship (PCR cycles to colour). The presenter uses the purpose config as well as the binning calculation code in the labware creator. And the binning ranges and PCR cycles could change so made sense it be in one place.
I get the point though.

Reply:
Ah OK, fair enough!